### PR TITLE
add a notice that the good first issue label is new and a WIP

### PR DESCRIPTION
**Pull request by clandry94**
Related to https://github.com/kubernetes/community/pull/2226 and https://github.com/kubernetes/community/issues/1959

I'm adding a notice that the `good first issue` label is still a WIP and not every repository is going to have them yet. New contributors might be discouraged when they see the fancy new `good first issue` label only to find that the repository they want to contribute to doesn't have any issues with that label.

### CODE REVIEW

**Comment by cblecker** [comment]
Can we add a comment to come back and review/remove this? Something like:
```
<!-- TODO: review for removal after 3 months or after the 1.12 release, whichever comes first -->
```

**Comment by clandry94** [comment]
updated!

**Comment by guineveresaenger** [comment]
Thank you @clandry94 - love that this was picked up on so quickly! Will leave the LGTM to @carolynvs.

File: contributors/guide/README.md [hunk 1]
```diff
@@ -74,6 +74,8 @@ You get the idea - if you ever see something you think should be fixed, you shou
 There are multiple repositories within the Kubernetes community and a full list of repositories can be found [here](https://github.com/kubernetes/).
 Each repository in the Kubernetes organization has beginner-friendly issues that provide a good first issue. For example, [kubernetes/kubernetes](https://git.k8s.io/kubernetes) has [help wanted](https://go.k8s.io/help-wanted) and [good first issue](https://github.com/kubernetes/kubernetes/labels/good%20first%20issue) labels for issues that should not need deep knowledge of the system.
 The `good first issue` label indicates that members have committed to providing extra assistance for new contributors.
+<!-- TODO: review removing this note after 3 momnths or after the 1.12 release -->
```
Comment (nikhita) [inline contributors/guide/README.md]: nit: s/momnths/months

Comment (cblecker) [reply contributors/guide/README.md]: Good catch!

**Comment by carolynvs** [comment]
I've got my lgtm finger prepped and ready to execute as soon as the typo is resolved. 😇

**Comment by clandry94** [comment]
Fire away with the lgtm @carolynvs!  ✅

**Comment by nikhita** [comment]
yay, congratulations on getting your first PR in to kubernetes!! :tada:

**Comment by carolynvs** [comment]
@clandry94 Thanks for turning that around so quickly! 💖

**Pull request merged**